src/kinesis.ts

```diff
@@ -89,7 +84,11 @@ export class KinesisEventHandler<Event, Context> {
             eventId: record.eventID,
           });
 
-          const parsedEvent = this.config.parseEvent(record.kinesis.data);
+          const parsedEvent = this.config.parseEvent(
+            JSON.parse(
+              Buffer.from(record.kinesis.data, 'base64').toString('utf8'),
+            ) as Record<string, unknown>,
```

Comment:
I think assuming base 64 is correct here, since that's a built-in Kinesis behavior. But, I don't think we should assume the message is JSON. That's not a Kinesis assumption.

Reply:
Gotcha, I think technically it's always a JSON object, but I get the point about aligning with the other abstractions already present. I don't think there's ever been the first occurrence where we don't just directly call `JSON.parse`, right?